app/src/sharedTest/java/org/oppia/app/profile/AdminPinActivityTest.kt

```diff
@@ -110,6 +112,86 @@ class AdminPinActivityTest {
     }
   }
 
+  @Test
+  fun testAdminPinActivity_rotateScreen_inputPin_inputConfirmPin_clickSubmit_checkOpensAddProfileActivity() {
+    ActivityScenario.launch<AdminPinActivity>(AdminPinActivity.createAdminPinActivityIntent(context, 0, -10710042)).use {
+      onView(ViewMatchers.isRoot()).perform(OrientationChangeAction.orientationLandscape())
```

Comment:
For better code clarity we can always import these functions directly.
`onView(isRoot()).perform(orientationLandscape())`